core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -160,9 +152,9 @@ object ConsumerGroupCommand {
     }
 
     protected def describeGroup(group: String) {
-      val configs = parseConfigs(opts)
```

Comment:
I too had the same thought, but then when we say that 0.9.0 clients won't work with 0.8.x brokers, I do not think we should be concerned about this.